src/main/java/frc/robot/subsystems/lifter/LifterStuff.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.lifter;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.signal.InputSignal;
+
+public record LifterStuff(
```

Comment:
Webster's dictionary states that stuff is: "materials, supplies, or equipment used in various activities"